cirq/ops/gate_operation.py

```diff
@@ -101,6 +101,9 @@ def _decompose_(self) -> op_tree.OP_TREE:
                                                     self.qubits,
                                                     NotImplemented)
 
+    def _pauli_expansion_(self) -> value.LinearDict[str]:
+        return protocols.pauli_expansion(self.gate)
+
```

Comment:
if you are bored, a lot of these delegates use the private _gate versus the property gate.  Would be nice to make consistent.

Reply:
Made them use the property instead of the private field. Thanks for spotting this.